cvat/apps/engine/management/commands/export_completed_annotations.py

```diff
@@ -65,7 +65,7 @@ def handle(self, *args, **options):
 
 
 def dump_annotations(task, dump_folder, overwrite=False):
-    if task.assignee.username != 'bot':
+    if task.assignee is not None and task.assignee.username != 'bot':
```

Comment:
should have had this bevore but maybe return if assignee is None?

Reply:
This is a completed task. Shouldn't it be dumped? The rest of the code does not depend on the assignee anyway.

Reply:
I mean this is a complete edge case, but I dont think we should export something if we dont know who annotated it